### PR TITLE
test: fixes on test-campaign.sh

### PR DESCRIPTION
**Pull request by chavafg**
This PR: 

1. adds a verification to check if `$GOPATH` environment variable is set
2. fixes number of tests reported.
3. adds `make install`  before running docker tests
4. adds sudo for running docker integration tests.

### CODE REVIEW

**Comment by gorozco1** [comment]
lgtm

**Comment by dlespiau** [comment]
lgtm

**Comment by chavafg** [comment]
qa-failed

**Comment by chavafg** [comment]
qa-passed

**Comment by chavafg** [comment]
qa-passed

**Comment by dlespiau** [comment]
@GabyCT could you ack this PR? seems like pullapprove got confused again.

File: tests/helpers/test-campaign.sh [hunk 1]
```diff
@@ -135,6 +139,12 @@ function run_make(){
 	( exit "${PIPESTATUS[0]}" ) && MAKE_RC=0
 }
 
+function run_make_install(){
+	echo "make execution"
+	sudo make install 2>&1 | tee "$MAKE_INSTALL_LOG_FILE"
+	( exit "${PIPESTATUS[0]}" ) && MAKE_INSTALL_RC=0
```
Comment (jodh-intel) [inline tests/helpers/test-campaign.sh]: neat! :)

**Comment by jodh-intel** [comment]
lgtm.

**Comment by gorozco1** [comment]
lgtm

**Comment by dlespiau** [comment]
This still needs approval from @GabyCT because pullapprove is stuck on that.

**Comment by GabyCT** [comment]
qa-passed

**Comment by dlespiau** [comment]
Thanks!

**Pull request merged**